docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+Additional capability was added into the `nuobackup` script allowing the removal of these jobs in [Helm Charts v3.1.0](https://github.com/nuodb/nuodb-helm-charts/releases/tag/v3.1.0).
+
+[2] - For deployments that contain nonHC SMs, it's recommended that database in-place restore is done by manually starting database engines in sequential order. First start the HC SMs, then nonHC SMs and last the TEs.
+
+[3] - _Automatic archive initial import_ is recommended when restoring a backup set taken from a different environment.
+
+[4] - _Archive seed restore_ is not deterministic. Once the restore request is created, the first SM that starts or is restarted will perform the restore of its archive.
+
+For detailed information about features, enhancements, and fixed problems, please check [NuoDB Release Notes](https://doc.nuodb.com/nuodb/latest/release-notes/) and [NuoDB Helm Charts Release Notes](https://github.com/nuodb/nuodb-helm-charts/releases)
+
+## Backup and Restore Mechanisms
+
+### Scheduled online database backups
+
+Multiple backup cron jobs are scheduled as part of the [database](../stable/database) chart deployment by default.
+The backup schedules can be customized to meet different backup and recovery objectives.
+They are exposed as Helm chart values in particular:
+- `database.hotCopy.fullSchedule` - configures the schedule of _full_ backup Kubernetes job
+- `database.hotCopy.incrementalSchedule` - configures the schedule of _incremental_ backup Kubernetes job
+- `database.hotcopy.journalBackup.intervalMinutes` - configures the schedule of _journal_ backup Kubernetes job. _journal_ hot copy must be explicitly enabled using `database.hotcopy.journalBackup.enabled`.
+
+[Cron expression format](https://en.wikipedia.org/wiki/Cron) documents the format of the first two schedules listed above.
+
+An HC SM has a backup volume attached and is selected for hot copy operation during a scheduled backup.
+Each _full_ hot copy will create a backup set with the same name in the backup root directory (by default `/var/opt/nuodb/backup`) on all HC SMs in a single database and a single backup group.
+By default, all HC SMs in a single Kubernetes cluster are part of one backup group.
+This can be adjusted by changing the `database.hotcopy.backupGroup` value for multiple Kubernetes clusters in multi-cluster deployment.
+_Incremental_ and _journal_ hot copies are always stored in the current backup set if one exists.
+If there is no current backup set for some of the archives served by a HC SM when _incremental_ or _journal_ backup is requested, a _full_ hot copy is triggered automatically.
+The `nuobackup` script is made available as a configMap and is used by all backup cron jobs to send a hot copy request of the specified type to the HC SMs.
+The _Full_ hot copy result is recorded into the NuoDB Admin domain key-value (KV) store upon successful finish.
```

Comment:
This seems to answer my questions above, but perhaps this could be reworded to make it clearer. Also, should we include the KV key format, or `nuocmd` invocation that can be used to obtain the current/latest full backup?

Reply:
Good point! I would suggest including the command wrappers rather than the KV prefixes.
The current backup set is available via `nuodocker get current backup --db-name <db>`.
The latest successful backup set is available via `nuobackup --type report-latest --db-name <db> --group <backup group>`.